build/Rulesets/Analyzers.ruleset

```diff
@@ -18,4 +18,9 @@
     <Rule Id="CA1720" Action="None" />
   </Rules>
 
+  <!-- TODO: Remove the below suppression once https://github.com/dotnet/roslyn-analyzers/issues/965 has been fixed. -->
```

Comment:
I have suppressed CA1305 due to 300+ violations of this rule. We should remove this suppression once https://github.com/dotnet/roslyn-analyzers/issues/965 is fixed.